pkg/resource/podtemplatespec.go

```diff
@@ -15,7 +15,7 @@ import (
 
 	configapiv1 "github.com/openshift/api/config/v1"
 	configlisters "github.com/openshift/client-go/config/listers/config/v1"
-	"github.com/openshift/cluster-image-registry-operator/pkg/apis/imageregistry/v1"
+	v1 "github.com/openshift/cluster-image-registry-operator/pkg/apis/imageregistry/v1"
```

Comment:
this seems redundant, your ide probably did it autonomously.

Reply:
yep. vscode's Golang extension seems to do this if multiple packages have the same name.